readme.md

```diff
@@ -53,7 +53,7 @@ body{
 }
 ```
 
-> Your answer...
+> The color red is in quotations so just delete those and it will work.
```

Comment:
:+1: We need to delimit the property and value with a colon, instead of "equals sign".